domenetjenester/arbeidsforhold/src/main/java/no/nav/foreldrepenger/domene/iay/modell/InntektsmeldingBuilder.java

```diff
@@ -176,6 +177,12 @@ public InntektsmeldingBuilder medRefusjon(BigDecimal verdi, LocalDate opphører)
         return this;
     }
 
+    public InntektsmeldingBuilder medRefusjonsperioder(List<Refusjon> endringerRefusjon) {
```

Comment:
Trenger denne for builder i tester

Reply:
Du kan ikke bare bruke metoden som finnes? leggTil(Refusjon refusjon)

Reply:
ah, så det fantes noe. Jeg fant den ikke. Bruker den 👍